homework-submission/react-hyf/react-hyf-server/server.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const app = express();
+
+app.use(express.static("/home/exilon/HYF/react/react-hyf/react-hyf-client/public"));
+
+app.get("/", (req, res) => {
+  res.sendFile(path.join(__dirname+'/react-hyf-client/build/index.html'));
```

Comment:
No need for this `get` method since we have the statics doing the exact same thing as here :)